core/include/ldml/keyman_core_ldml.ts

```diff
@@ -564,8 +564,16 @@ class Constants {
   /** maximum count of markers (not including 'any') */
   readonly marker_max_count    = this.marker_max_index - this.marker_min_index + 1;
 
+  /**
+   * List of versions to treat as if they were the 'latest' version.
+   */
+  treatAsLatest(version: string): boolean {
+    return cldrTreatAsLatest.has(version);
+  }
 };
 
+const cldrTreatAsLatest: Set<string> = new Set(['45', '46.1', '47']);
+
```

Comment:
I don't really understand what this is doing? And why `46.1` vs `46`, does this mean that `46` is not 'treated as latest' even though `cldr_version_latest` == '46'?

Reply:
it treats imports of 45/ and 47/ as if they were 46/ (as the data is all the same).  46.1 may not be needed here. 

I didn't import 46.1 as it's not a major release.

Reply:
I think we need a code comment to clarify why there are older and newer versions listed here -- particularly for future maintainers.